lib/string.fz

```diff
@@ -548,6 +592,19 @@ strings is
       utf8 Sequence u8 is a.utf8 ++ b.utf8
 
 
+  # Takes a Sequence of string and concatenates its elements, while adding the separator
```

Comment:
sequence of strings

I think I prefer not using _code language_ but that is certainly a matter of taste.